assets/studio_lenzing.css

```diff
@@ -0,0 +1,31 @@
+body {
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 700;
+}
+
+.topnav {
```

Comment:
use kebab case (word separation using ` - `) for css classes or learn BEM (http://getbem.com/)

Reply:
This is noted.